service/history/nDCTransactionMgr_test.go

```diff
@@ -173,7 +182,13 @@ func (s *nDCTransactionMgrSuite) TestBackfillWorkflow_CheckDB_NotCurrent() {
 	defer mutableState.AssertExpectations(s.T())
 	var releaseFn releaseWorkflowExecutionFunc = func(error) { releaseCalled = true }
 
-	workflowEvents := &persistence.WorkflowEvents{}
+	workflowEvents := &persistence.WorkflowEvents{
+		Events: []*shared.HistoryEvent{{
+			EventType: common.EventTypePtr(shared.EventTypeWorkflowExecutionSignaled),
+		}},
+		DomainID:   domainID,
+		WorkflowID: workflowID,
+	}
```

Comment:
missing a ?
`context.On("reapplyEvents", mock.Anything, workflowEvents).Return(nil).Times(1)`

Reply:
The reapply moved executionContext. no need to verify it from here